.github/ISSUE_TEMPLATE/DesignDoc.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Design Doc tracking issue
+about: A standard template to follow when leading a project
+labels: Daily, NewFeature
+---
+
+## Proposal
+
+## Tasks
+
+- [ ] Post **Proposal (Problem/Solution)** in #expensify-open-source
+- [ ] Wait at least one full business day, and until the post has a majority (2/3) of positive reactions (👍)
```

Comment:
Could be nice to recommend a minimum number of reactions, maybe 10 - I think posts in #expensify-open-source **can** go unnoticed sometimes so i think we should get at least 10 reactions to consider moving forward with a P/S
- Also it could be good to wait for a minimum number of expensify employee reactions, no?

Reply:
Hmm, yeah that's a fair point. At the same time, any number we pick is arbitrary. Internally the way we'd solve this is say, make sure that two of graduates/project-managers/etc. etc. review the doc. So ideally we could extend that same concept here.

Reply:
Though I'm getting a bit ahead of myself. To be fair, we don't really have an internal standard in #whatsnext as to whether you can or can't move forward. It's really kind of the opposite, did anyone vote to stop your idea from moving forward. The thumbs up are really the easy part.

Reply:
I do not see what problem setting an arbitrary number of reactions solves at this point. The goal of this checklist is to make sure that vendors who we've already asked to do a Design Doc are following our process more rigorously.